templates/accounts/email_reset_form.html

```diff
@@ -4,7 +4,13 @@
 
 {% block content %}
 
-<h1>E-mail reset</h1>
+{% if required %}
```

Comment:
Maybe we can use `django.contrib.messages` here so the formatting is the "standard" in Freesound.
It's a matter of importing `from django.contrib import messages` in `middleware` and adding a message before redirecting to email reset page in `UpdateEmailHandler`: 
```
messages.add_message(request, messages.INFO, "We have identified that some emails that we have sent to you didn't go through, thus it appears that your email address is not valid. Please update your email address to a working one to continue using Freesound")
```